mars/dataframe/expressions/core.py

```diff
@@ -118,6 +118,11 @@ class DataFrameOperand(Operand):
     def object_type(self):
         return self._object_type
 
+    @classmethod
+    def execute(cls, ctx, op):
+        chunk = op.outputs[0]
+        ctx[chunk.key] = chunk.op.data
```

Comment:
this is `DataFrameDataSource`'s execution, not the base class's. And now it's better not to refactor DataFrame, open a new pull request to do it.

Reply:
Hi, should I firstly open a issue to write it down?

Reply:
As a part of refactor, just open a pull request.